draft-ietf-quic-transport.md

```diff
@@ -4285,7 +4267,7 @@ Destination Connection ID Length:
 Destination Connection ID:
 
 : The Destination Connection ID field follows the Destination Connection ID
-  Length field and is between 0 and 20 bytes in length.
+  Length field, which indicates the length of this field.
```

Comment:
By not explicitly calling out the length, we don't have to repeat the discussion about v1 vs. invariant.